misc/dbt-materialize/MAINTAINERS.md

```diff
@@ -29,28 +29,31 @@
 5. Launch `materialized` in a different terminal:
 
    ```
-   brew install materialize/materialize/materialized
+   brew install MaterializeInc/materialize/materialized
+
    materialized
    ```
 
-6. Run tests:
+6. Install `pytest-dbt-adapter` and run tests:
```

Comment:
Are these instructions a bit useless now that you need a running Kafka service to run the tests?

Reply:
Thanks for the reminder, @dataders! We have this queued as a follow-up task from #12141!